scss/_color_use_cases.scss

```diff
@@ -0,0 +1 @@
+@include oColorsSetUseCase(o-typography-aside-text, text, 'grey-tint4');
```

Comment:
Probably better to use the `cold-1` name as that's the  Next way of calling it, it seems :P